django/contrib/auth/hashers.py

```diff
@@ -774,9 +776,11 @@ def salt(self):
 
     def encode(self, password, salt):
         crypt = self._load_library()
-        assert len(salt) == 2
+        if len(salt) != 2:
+            raise ValueError('Salt must contain two characters, but got "{}".'.format(salt))
```

Comment:
It looks like you used `salt` instead of its length. I think the length of `salt` will be the better option here.

Reply:
And also wrap the error message at 79 chars. You can also use `fstring`.